CommunityFridgeMapApi/tests/unit/test_tag_is_valid.py

```diff
@@ -0,0 +1,25 @@
+from multiprocessing import connection
```

Comment:
I think we can delete this file? Since you a already test `is_valid_tag_name` in the test_tag file

Reply:
I deleted that from my branch before the commit and push. I'll check again!